backend/tests/test_schema/test_inital_data.py

```diff
@@ -0,0 +1,21 @@
+from tests.test_schema.helpers import PytestHelpers
+from sqlalchemy.orm import Session
+from app.initial_data import populate_initial_data
+
+
+def test_initial_data_populates_tables(engine):
+    helpers = PytestHelpers(engine)
+    helpers.add_alembic()
+
+    with Session(engine) as db:
+        populate_initial_data(db)
+        db.commit()
```

Comment:
maybe that `flush()` suggestion was more than a nit...?

Reply:
This results in the tests failing:

```
_______________________________________ test_initial_data_populates_tables ________________________________________

engine = Engine(postgresql://navigator:***@backend_db:5432/navigator_test)

    def test_initial_data_populates_tables(engine):
        helpers = PytestHelpers(engine)
        helpers.add_alembic()
    
        with Session(engine) as db:
            populate_initial_data(db)
            db.flush()
    
        with Session(engine) as db:
            geo_count = db.execute("SELECT count(*) FROM geography;").scalar()
            language_count = db.execute("SELECT count(*) FROM language;").scalar()
            geo_stats_count = db.execute("SELECT count(*) FROM geo_statistics;").scalar()
    
>       assert geo_count == 210
E       assert 0 == 210
```

Reply:
sorted